crates/composefs-setup-root/src/main.rs

```diff
@@ -116,6 +120,9 @@ fn bind_mount(fd: impl AsFd, path: &str) -> rustix::io::Result<OwnedFd> {
             | OpenTreeFlags::OPEN_TREE_CLOEXEC
             | OpenTreeFlags::AT_EMPTY_PATH,
     )
+    .inspect_err(|_| {
+        eprintln!("Open tree failed for {path}");
```

Comment:
Interesting.  I don't know this pattern.  I guess it makes sense with this not being a library...